src/screens/EncounterHomeScreen.tsx

```diff
@@ -62,6 +80,14 @@ export const EncounterHomeScreen = ({ navigation, route }: any) => {
     });
   }, [navigation]);
 
+  const onPressToCategory = useCallback(() => {
+    navigation.navigate(ScreenName.Index, {
+      query: QUERY_TYPES.POINTS_OF_INTEREST,
+      queryVariables: { categoryId: encounterCategoryId },
```

Comment:
do we miss `limit: 15, order: 'name_ASC'` here, like in `parseCategories` in `listItemParser.js`?

Reply:
it is probably better to add it here as well. so i've added it ;)